.github/.cSpellWords.txt

```diff
@@ -1,12 +1,19 @@
+ASDFLKJ
```

Comment:
Why are these nonsense combinations in the dictionary??

Reply:
spellchecker was picking them up from unit tests